app/src/main/java/ru/yandex/practicum/contacts/presentation/base/ListDiffInterface.java

```diff
@@ -0,0 +1,6 @@
+package ru.yandex.practicum.contacts.presentation.base;
+
+public interface ListDiffInterface<T> {
+    public boolean theSameAs(T newItem);
```

Comment:
🍏 Модифайер public не обязателен для методов интерфейсов:

```suggestion
    boolean theSameAs(T newItem);
```